tools/cldr-apps/js/src/views/GearMenu.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <ul>
+    <li v-if="isAdmin"><a href="#admin">Admin Panel</a></li>
+    <li v-if="loggedIn" class="section-header">My Account</li>
+    <li>
+      <ul>
+        <li><a href="#account">Account Settings</a></li>
+      </ul>
+    </li>
+    <li v-if="loggedIn">
+      <ul>
+        <li>
+          <a href="lock.jsp" target="_blank">Lock (Disable) My Account </a>
+        </li>
+      </ul>
+    </li>
+    <li v-if="loggedIn" class="section-header">My Votes</li>
+    <li v-if="canImportOldVotes">
+      <ul>
+        <li><a href="#oldvotes">Import Old Votes </a></li>
+      </ul>
+    </li>
+    <li v-if="loggedIn">
+      <ul>
+        <li><a v-bind:href="recentActivityUrl">Recent Activity</a></li>
+      </ul>
+    </li>
+    <li v-if="loggedIn">
+      <ul>
+        <li>
+          <a
+            href="upload.jsp?a=/cldr-apps/survey&amp;s=w5nGTzkdT_bY9LnApVazwpX"
+            target="_blank"
+            >Upload XML
+          </a>
+        </li>
+      </ul>
+    </li>
+    <li v-if="loggedIn" class="section-header">My Organization ({{ org }})</li>
+    <li v-if="loggedIn">
+      <ul>
+        <li><a href="#vsummary">Priority Items Summary (slow)</a></li>
+      </ul>
+    </li>
+    <li v-if="loggedIn">
+      <ul>
+        <li>
+          <a href="#list_users">List {{ org }} Users</a>
+        </li>
+      </ul>
+    </li>
+    <li v-if="loggedIn">
+      <ul>
+        <li><a href="#forum_participation">Forum Participation</a></li>
+      </ul>
+    </li>
+    <li v-if="loggedIn">
+      <ul>
+        <li><a href="#vetting_participation">Vetting Participation</a></li>
+      </ul>
+    </li>
+    <li v-if="accountLocked" class="emphatic">
+      LOCKED: Note: your account is currently locked
+    </li>
+    <li v-if="loggedIn" class="section-header">Forum</li>
+    <li v-if="loggedIn">
+      <ul>
+        <li>
+          <a href="#flagged"
+            ><img
+              src="flag.png"
+              alt="flag"
+              title="flag.png"
+              border="0"
+            />Flagged Items
+          </a>
+        </li>
+      </ul>
+    </li>
+    <li v-if="loggedIn">
+      <ul>
+        <li><a href="#mail">Notifications (SMOKETEST ONLY)</a></li>
+      </ul>
+    </li>
+    <li v-if="isAdmin">
+      <ul>
+        <li><a href="#bulk_close_posts">Bulk Close Posts</a></li>
+      </ul>
+    </li>
+    <li class="section-header">Informational</li>
+    <li>
+      <ul>
+        <li><a href="#statistics">Statistics</a></li>
+      </ul>
+    </li>
+    <li>
+      <ul>
+        <li><a href="#about">About</a></li>
+      </ul>
+    </li>
+    <li>
+      <ul>
+        <li><a href="#lookup">Look up a code or xpath</a></li>
+      </ul>
+    </li>
+    <li v-if="isTC">
+      <ul>
+        <li><a href="#error_subtypes">Error Subtypes</a></li>
+      </ul>
+    </li>
+  </ul>
+</template>
+
+<script>
+import * as cldrStatus from "../../../src/main/webapp/js/esm/cldrStatus.js";
```

Comment:
not needed. It's already `$cldrOpts.cldrStatus`

Reply:
I kind of like an explicit "import" statement anyway -- more modular